src/app/components/datetime-picker/datetime-picker.component.ts

```diff
@@ -64,18 +74,41 @@ export class DatetimePickerComponent implements OnInit {
     this.datePickerOpen = false;
     this.datePickerOpenChange.emit(this.datePickerOpen);
 
-    if (event.detail.data && event.detail.role === 'confirm' || this.resetable) {
+    if ((event.detail.data && event.detail.role === 'confirm') || this.resetable) {
       this.dateTimeChange.emit(event.detail.data);
     }
   }
 
+  updateDateOnWeb(event: string) {
```

Comment:
Kanskje bruke et annet navn enn "event" for å beskrive mer presist hva denne parameteren inneholder?